src/client/sound/al_extensions.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+Minetest
+Copyright (C) 2023 DS
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; ifnot, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#include "al_extensions.h"
+
+#include "settings.h"
+#include "util/string.h"
+#include <unordered_set>
+
+namespace sound {
+
+ALExtensions::ALExtensions(const ALCdevice *deviceHandle [[maybe_unused]])
```

Comment:
I thought attributes go next to the type

Reply:
I used to have it left of the type, but the old gcc 7 in the CI could not parse it. This seems to work.